src/lms-harmonizer/edfi_lms_harmonizer/runner.py

```diff
@@ -43,13 +44,15 @@ def run(arguments: MainArguments) -> None:
     exceptions_report_directory = arguments.exceptions_report_directory
 
     harmonize_users(arguments.engine, adapter)
-    harmonize_sections(arguments.engine, adapter)
-    harmonize_assignments(arguments.engine, adapter)
-    harmonize_assignment_submissions(arguments.engine, adapter)
 
-    if exceptions_report_directory is not None:
-        create_exception_reports(adapter, exceptions_report_directory)
+    if arguments.engine == DB_ENGINE.MSSQL:
```

Comment:
Temporary. A bunch of test assertions might fail, but the app itself won't be the cause.

Reply:
Ok